_docs/demo/helloworld.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: page
+title: Hello World!
+permalink: /docs/demo/helloworld/
+category: Demo
+order: 2
+---
+
+This demo starts two versions of a helloworld microservice each with two
+versions, in order to demonstrate how Amalgam8 can be used to split
+incoming traffic between the two versions. You can define the proportion of
+traffic to each microservice as a percentage.
+
+## Deploy and Scale the App
+
+The commands to deploy the helloworld demo application for different
+environments are as follows:
+
+_Docker Compose_
+  
+```bash
+docker-compose -f examples/docker-helloworld.yaml up -d
+```
+
+Scale the individual versions as follows:
+
+```bash
+docker-compose -f examples/docker-helloworld.yaml scale helloworld-v1=2
+docker-compose -f examples/docker-helloworld.yaml scale helloworld-v2=2
+```
+
+_Kubernetes_ on localhost or on Google Cloud
+
+```bash
+kubectl create -f examples/k8s-helloworld.yaml
+```
+
+The above command automatically launches two instances of each version of
+`helloworld`.
+
+_IBM Bluemix_
+
+1. Create Bluemix routes (DNS names) for the registry, controller and the helloworld app's gateway:  
+
+   ```bash
+   cf create-route <your bluemix space> mybluemix.net -n <your helloworld route>
```

Comment:
Same comment as the other file.  "bluemix" is the unified CLI for Bluemix.  So change "cf create-route...." to "bluemix cf create-route...."